db/migrate/20181004013839_add_composite_index_to_news_coin_mention.rb

```diff
@@ -0,0 +1,22 @@
+class AddCompositeIndexToNewsCoinMention < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def change
+    reversible do |dir|
+      dir.up do
+        # remove duplicates
+        execute <<-SQL
+          DELETE FROM news_coin_mentions AS T1 
```

Comment:
@anthotsang Just a heads up that I personally prefer to separate schema migrations from data migrations; schema migrations live in `db/migrate` and data migrations live in `lib/tasks/data_migrations`.

The rationale for this is that it might be useful to have the data migration script separate from the schema migration so we can rerun it later if needed.

In this case though, I think it's fine to keep the data migration with this schema migration since it should be a one time task (as your unique index should prevent more duplicates in the future).